examples/ultralytics-yolov3/deepsparse-yolov3.ipynb

```diff
@@ -0,0 +1,308 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<sub>&copy; 2021-present Neuralmagic, Inc. // [Neural Magic Legal](https://neuralmagic.com/legal)</sub> \n",
+    "\n",
+    "# Running YOLO Models on the DeepSparse Engine\n",
+    "\n",
+    "In [this blog post](https://neuralmagic.com/blog/benchmark-yolov3-on-cpus-with-deepsparse/) the Neural Magic team describes how achieving real time inference speeds on [Ultralytic's YOLOv3](https://github.com/ultralytics/yolov5) using CPUs is possible. This notebook provides a step-by-step walkthrough for using the [DeepSparse](https://github.com/neuralmagic/deepsparse) inference engine to attain and benchmark these speeds.\n",
+    "\n",
+    "This notebook runs pre-sparsified models downloaded from the [SparseZoo](https://github.com/neuralmagic/sparsezoo). For information on creating custom sparsified YOLO models with [SparseML](https://github.com/neuralmagic/sparseml), check out the [SparseML-Ultralytics integration documentation](https://github.com/neuralmagic/sparseml/blob/main/integrations/ultralytics/README.md).\n",
+    "\n",
+    "\n",
+    "In this notebook, you will:\n",
+    "- Compile a model with the DeepSparse engine\n",
+    "- Use the DeepSparse engine to run a sample inferences with YOLOv3 and annotate the output\n",
```

Comment:
```suggestion
    "- Use the DeepSparse Engine to run a sample inference with YOLOv3 and annotate the output\n",
```

Reply:
will the user be running more than one inference or just one reference? my suggested edit assumes one. please update though if it's more than one